src/interfaces/python/swig_typemaps.i

```diff
@@ -165,51 +165,9 @@ static int is_pysparse_matrix(PyObject* obj, int typecode)
          ) ? 1 : 0;
 }
 
-static int is_pystring_list(PyObject* obj, int typecode)
-{
```

Comment:
no need for this method, we can depend on std::vector typemaps

Reply:
great!